src/components/TimeBasedLineChart.js

```diff
@@ -227,6 +226,7 @@ const TimeBasedLineChart = React.createClass({
       shadeBelowZero: false,
       shadeFutureOnGraph: true,
       showBreakPoint: true,
+      showZeroLine: false,
```

Comment:
Yes you are correct. I already released but I can bump the version and release again.